connection.js

```diff
@@ -24,14 +23,14 @@ let ignorable_notifications = {
 };
 
 class Connection {
-    constructor(io_client) {{{
-        let prefix = (config.insecure ? 'http://' : 'https://') + config.host + ':' + config.port;
+    constructor(io_client) {
+        const prefix = (config.insecure ? `http://` : `https://`) + `${config.host}:${config.port}`;
```

Comment:
Why use string interpolation if there are no variables in `http://` and `https://`?
Also possible:
`http${config.insecure ? '' : 's'}://${config.host}:${config.port}`